gcore/gdal.h

```diff
@@ -2552,6 +2552,12 @@ int CPL_DLL GDALDimensionSetIndexingVariable(GDALDimensionH hDim,
                                              GDALMDArrayH hArray);
 bool CPL_DLL GDALDimensionRename(GDALDimensionH hDim, const char *pszNewName);
 
+CPLErr CPL_DLL GDALRasterInterpolateAtPoint(GDALRasterBandH hBand,
```

Comment:
this should be moved upper in the file, with the other functions that take GDALRasterBandH as their first arguement

Reply:
done